core/src/main/java/net/opentsdb/query/processor/merge/Merger.java

```diff
@@ -146,7 +146,7 @@ public Merger(final QueryNodeFactory factory,
                 .build();
       }
     }
-
+    LOG.info("Expect to have results: " + results);
```

Comment:
debug

Reply:
want to have it under info for now. doesn't generate too much and will be good to have to visually test the split code

Reply:
Double check the log configs, I think it's set to debug for prod too :)